packages/confirm-dialog/src/styles/vaadin-confirm-dialog-overlay-base-styles.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2018 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css } from 'lit';
+import { dialogOverlayBase } from '@vaadin/dialog/src/styles/vaadin-dialog-overlay-base-styles.js';
+import { overlayStyles } from '@vaadin/overlay/src/styles/vaadin-overlay-base-styles.js';
+
+const confirmDialogOverlay = css`
+  @layer base {
+    :host {
+      --vaadin-dialog-min-width: var(--vaadin-confirm-dialog-min-width, 15em);
+      --vaadin-dialog-max-width: var(--vaadin-confirm-dialog-max-width, 25em);
+    }
+
+    ::slotted([slot='header']) {
+      display: contents;
+      font: inherit !important;
```

Comment:
It looks like usage of `!important` here could be the problem with base-layer reset.

Reply:
I think it would be acceptable, and would make sense.

Reply:
I’ll introduce the breaking change (for the existing core styles) and updated Lumo styles in a new PR.

Reply:
https://github.com/vaadin/web-components/pull/9570

Reply:
@web-padawan, could you do the rebase/merge?